### PR TITLE
routes, swagger-ui, devops

### PR DESCRIPTION
**Pull request by brianfunk**
Create Non-Commercial Routes #9
Create Outfitter Routes #10
Create mock output JSON for documentation #13
Create SwaggerUI/OpenAPI JSON for Non-Commercial Routes #19
Create SwaggerUI/OpenAPI JSON for Outfitter Routes #23 
Specify repo in travis deploy #38 
Rename /public to /docs #39
Update tags in package.json #40
Gemnasium setup #41

### CODE REVIEW

**Comment by lpendem** [comment]
naming convention is not consistent with outfitters; it referred as outfitters, temp-outfitters, and outfitters and guides in API and Documentation.

File: test/data/outfitters.get.all.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          "emailAddress": "test@email.org",
+          "mailingAddress": "ON ANW 0953",
+          "mailingCity": "ALBANY",
+          "mailingState": "OR",
+          "mailingZIP": 97321,
+          "organizationName": "COOLEY RIVER EXPEDITIONS",
+          "website": null,
+          "orgType": "Corporation"
+        },
+        "outfitters-fields":{
```
Comment (cameronwolf) [inline test/data/outfitters.get.all.json]: Needs to be 'temp-outfitter-fields' not 'outfitters-fields' to match with schema.

File: test/data/outfitters.get.all.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          "insuranceCertificate": {
```
Comment (cameronwolf) [inline test/data/outfitters.get.all.json]: This should more closely match the response of get/id. insuranceCertificate, goodStandingEvidence, and operatingPlan are structured differently than they are in the get/id response.

File: test/data/outfitters.get.id.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      "emailAddress": "test@email.org",
+      "mailingAddress": "ON ANW 0953",
+      "mailingCity": "ALBANY",
+      "mailingState": "OR",
+      "mailingZIP": 97321,
+      "organizationName": "COOLEY RIVER EXPEDITIONS",
+      "website": null,
+      "orgType": "Limited Liability Company"
+    },
+    "temp-outfitter-fields": {
```
Comment (lpendem) [inline test/data/outfitters.get.id.json]: temp-outfitter-fields matches with the mapping JSON.

File: test/data/outfitters.get.id.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      "activityDescription": "PROVIDING WHITEWATER OUTFITTING AND GUIDING ACTIVITIES ON NATIONAL FOREST LANDS",
+      "locationDescription": "string",
+      "startDateTime": "2013-01-12",
+      "endDateTime": "2013-01-19",
+      "insuranceCertificate": "insuranceCertificate.pdf",
+      "goodStandingEvidence": "goodStandingEvidence.pdf",
```
Comment (lpendem) [inline test/data/outfitters.get.id.json]: insuranceCertificate  should be of file type, goodStandingEvidence and operatingPlan fields will be added by the API, not from this POST request; these needs to be reviews with the client.

**Comment by janwilli** [comment]
Tested and approved.

**Pull request merged**